conu/utils/http_client.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from six.moves.urllib.parse import urlunsplit
+
+
+class HttpClient:
```

Comment:
Could this class be a child of requests.Session so that we don't have to reimplement all the get, post, put methods? Even if we want to keep the workflow:
```
.post("/bugzilla/create", data={}, ...)
```

Reply:
I'm not sure if I get it right. Can you look at the new commit?